kymatio/scattering2d/scattering2d.py

```diff
@@ -98,10 +98,10 @@ def __init__(self, J, shape, L=8, max_order=2, pre_pad=False):
     def build(self):
         self.M, self.N = self.shape
         self.modulus = Modulus()
-        self.pad = Pad(2**self.J, pre_pad = self.pre_pad)
+        self.M_padded, self.N_padded = compute_padding(self.M, self.N, self.J)
+        self.pad = Pad([(self.N_padded-self.N)//2,(self.N_padded-self.N+1)//2,(self.M_padded-self.M)//2,(self.M_padded-self.M+1)//2], pre_pad = self.pre_pad)
```

Comment:
remove spaces around keyword arguments, add spaces around operators

Reply:
Can you add a comment explaining what this does?
It looks to me like to pads equally on both sides if the amount of padding to add is an even number of pixels, and will add the extra pixel on the right and the bottom if it's an odd number of pixels. Is this correct? If yes, can you add a comment to that effect? If not, can you add a comment explaining what is correct?

Reply:
Fixed!